src/go.mod

```diff
@@ -3,8 +3,8 @@ module std
 go 1.15
 
 require (
+	github.com/cloudflare/circl v1.0.1-0.20201123155603-cd1efefc7f5f
```

Comment:
Not sure what the effect is of including dependencies here on applications using the standard Go library. In any case this will cause conflicts during major updates that update this file.

Reply:
Yes, this should also be checked.